src/client.ts

```diff
@@ -28,6 +28,23 @@ export interface ClientConfig {
   idleTimeout?: number;
   /** charset */
   charset?: string;
+  /** tls config */
+  tls?: TLSConfig;
+}
+
```

Comment:
I think this enum shoud be re-exported from `mod.ts`

https://github.com/denodrivers/mysql/blob/6904845798ab63a28b9159488a62ffcf1db15b52/mod.ts#L2